hooks/helmlint.sh

```diff
@@ -146,16 +146,26 @@ for file in "${files[@]}"; do
     linter_values_arg=""
   fi
 
+  # Add any additional values files compatible with the chart-testing tool ([docs](https://github.com/helm/chart-testing))
+  ci_values_args=()
+  for file in "$file_chart_path"/ci/*-values.yaml; do
+    if [ ! -e "$file" ];  then
```

Comment:
```suggestion
    if [ ! -f "$file" ];  then
```

`-f` is more precise. Otherwise, for example, a `ci/foo-values.yaml/` directory would be included.

Reply:
Thanks for catching this. Updated.